src/content/en/begin-with/custom-nodes.md

```
@@ -36,19 +39,19 @@ Basically, install from **ComfyUI Manager**.
 ![](https://gyazo.com/c0d8901537b65da709f9ba9d6e1a0055){gyazo=loop}
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)